dietpi/dietpi-letsencrypt

```diff
@@ -193,18 +183,17 @@ _EOF_
 
 			/etc/init.d/lighttpd force-reload
 
-
 		#------------------------------------------------------------------------------------------------------
 		# Nginx
-		elif (( $(ps aux | grep -ci -m1 '[n]ginx') )); then
+		elif pgrep -i '[n]ginx' &> /dev/null; then
```

Comment:
Interesting, seems ```pgrep``` automatically removes the process which checks for it, so its not added to the count:
Starting with ```[n]``` is not then needed, but no harm 😃 
```
root@DietPi:~# pgrep -i bash
2318
2402
root@DietPi:~# pgrep -i [b]ash
2318
2402
root@DietPi:~# ps aux | grep bash
root      1002  0.0  0.0   4308   580 pts/0    S+   15:19   0:00 grep --color=auto bash
root      2318  0.0  0.1   6132  4036 ?        Ss   13:36   0:00 -bash
root      2402  0.0  0.1   7144  5320 pts/0    Ss   13:37   0:00 -bash
root@DietPi:~# ps aux | grep [b]ash
root      2318  0.0  0.1   6132  4036 ?        Ss   13:36   0:00 -bash
root      2402  0.0  0.1   7144  5320 pts/0    Ss   13:37   0:00 -bash
```